### ISSUE TITLE
Allow disabling of cache anywhere in the query chain.

### ISSUE THREAD

**Issue by jetwes** [GeneaLabs/laravel-model-caching#94]
If there any any queries with ->disableCache() in your app and you would like to 
disable the cache globaly with the .env entry
`MODEL_CACHE_DISABLED=TRUE` 
you'll get plenty of Undefined method disableCache() errors.

Maybe - as an enhancement - you return an empty method disableCache() when the cache is disabled globally - so the queries will still work.

**Comment by mikebronner** [comment]
This is probably caused because you aren't calling `disabledCached()` on the model directly. It should only be called from the model, not within the query. For example, this is the intended way:

```php
$model->disableCache()->with('relationship')->where('name', $name)->get();
```

This following is **not the intended way** (but works for the moment just because that is the way the code is organized after moving all functionality into a single trait (which will change soon):
```php
$model->with('relationship')->where('name', $name)->disableCache()->get();
```

**Comment by mikebronner** [comment]
In version 0.2.43 `disableCache()` has been moved out of CachedBuilder, so it should force you to put it in the correct position now.

**Comment by mikebronner** [comment]
I have updated the readme to reflect this. I also will look into making it globally insertable anywhere in the query chain again.

**Comment by mikebronner** [comment]
Quick update: `disableCache()` is currently not behaving correctly. I hope to have a fix for this today.

**Comment by mikebronner** [comment]
This is now fixed in release 0.2.44.
